pages/linux/shred.md

```diff
@@ -0,0 +1,15 @@
+# shred
+
+> Shred - overwrite a file to hide its contents, and optionally delete it
+
+- Overwrite the specified FILE(s) repeatedly, in order to make it harder for even very expensive hardware probing to recover the data.
```

Comment:
I'd simplify this to say just "harder to recover the data."

Also, FILE should not be in all caps.